pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,44 @@
+//include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the {PublishingPortal}.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged in.
+* The module to be published has been imported to {ProductShortName}.
+* The metadata required for publishing has been applied to the module.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to publish.
++
+If you do not see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click  *Publish* next to the version of the module you want to publish.
++
+After you publish the module, you can access it at the following URL:
++
+`https://__PRODUCT_URL__/__VANITY_URL_FRAGMENT__`
++
+where, __PRODUCT_URL__ is the URL for the product, and __VANITY_URL_FRAGMENT__ is the vanity URL fragment for the module.
```

Comment:
Thanks for the note, @benradey!

@theashiot - I agree that today, it probably makes sense for this to state the automatically generated UUID rather than a vanity URL, especially since we're not covering that exact field in this procedure.

Shall we change it?

Reply:
Thanks @benradey and @adahms, changed the text to:

"https://__PRODUCT_URL__/__UUID__`

where, __PRODUCT_URL__ is the URL for the product, and __UUID__ is the auto-generated unique identifier for the module. "

what do you think?

Reply:
Sounds good to me!